CHANGELOG.md

```diff
@@ -5,13 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-
 ## Unreleased
 
 ### Added
 - `use_gpu` for PureSVD ([#229](https://github.com/MobileTeleSystems/RecTools/pull/229))
 - `from_params` method for models and `model_from_params` function ([#252](https://github.com/MobileTeleSystems/RecTools/pull/252))
+- `TorchRanker` ranker which calculates scores using torch. Supports GPU.
+- `Ranker` ranker protocol which unify rankers call.
+
+### Changed
 
+- `ImplicitRanker` `rank` method compatible with `Ranker` protocol. 
+`use_gpu` and `num_threads` params moved from `rank` method to `__init__`.
```

Comment:
If this is a separate line, there should be a bullet. If no, then it's better to put everything on the same line

Reply:
fixed